clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -91,6 +91,10 @@ Improvements to clang-query
 Improvements to clang-tidy
 --------------------------
 
+- It no longer processes declarations from system headers by default, greatly
```

Comment:
I think we should mention somewhere that downstream users might need to modify their own checks as this might change their behavior by default.